### DISCUSSION TITLE
Double autocompletebox dropdown

### DISCUSSION THREAD

**Discussion by gmol9knots** [AvaloniaUI/Avalonia#15988] [Q&A]
### Describe the bug

When typing in autocompletebox it sometimes creates two dropdowns

### To Reproduce

Create autocompletebox with a lot of items

### Expected behavior

just one dropdown

### Avalonia version

11.0.10

### OS

Windows

### Additional context

![image](https://github.com/AvaloniaUI/Avalonia/assets/108529890/8a0e5752-4f6d-47a2-b1c2-45a0e63cd9f5)

**Comment by timunie** [comment]
Never seen it. Looks like a custom style. Can you wire a minimum sample together and file it here?

**Comment by gmol9knots** [comment]
This is the only styling i have applied to the AutoCompleteBox i dont have any behaviours linked

```
<Style Selector="AutoCompleteBox">
	<!-- Default font size of the Application -->
	<Setter Property="IsTextCompletionEnabled" Value="True"/>
	<Setter Property="MinimumPrefixLength" Value="0"/>
</Style>
<Style Selector="AutoCompleteBox">
        <Setter Property="CornerRadius" Value="{DynamicResource CornerRadius}"/>
</Style>
<Style Selector="AutoCompleteBox.Expr">
        <Setter Property="Background" Value="{DynamicResource ExprColor}"/>
</Style>
<Style Selector="AutoCompleteBox.Director">
       <Setter Property="Background" Value="{DynamicResource DirectorColor}"/>
</Style>
<Style Selector="AutoCompleteBox.Mandatory">
       <Setter Property="Background" Value="{DynamicResource MandatoryColor}"/>
</Style>
```

**Comment by timunie** [comment]
Can't reproduce. Please file a sample App
